src/utils/get.js

```diff
@@ -12,17 +12,29 @@ import { isObject } from './object'
  * @return {*}
  */
 export default (obj, path, defaultValue = null) => {
+  // Handle aray of path values
+  path = isArray(path) ? path.join('.') : path || ''
```

Comment:
I think we don't need the `''` fallback here since the early exit below will handle it anyway.

Reply:
Fixed